source/localizable/partner_sku_reviews/wordpress_widgets.html.md.erb

```diff
@@ -0,0 +1,103 @@
+
+> ##### Note
+The widgets are displayed only on product pages and if there is at least one review for the product.
+
+## Add Widgets from UI
+
+You can add the widgets in any available [widget area](https://codex.wordpress.org/Glossary#Widget_Area) your theme supports.
+
+### Theme customizer
+
+If your theme supports it, you can use the theme customizer.
+
+* Make sure you are logged in the admin panel and your user has a role that can edit the theme.
+* Navigate to a product page in the front.
+* In the upper left menu you should see a `Customize` button.
+* Select the `Widgets` in the customizer menu on the left.
+* A list with the available widget areas should appear.
+* Select the area that you want the product reviews widget to be displayed and then select `Add a Widget`.
+* The two <%= settings.site_name.capitalize %> product reviews widgets should appear in the list with the available widgets.
+* Finally select the widget and click `Publish`.
+
+<%= image_tag "/assets/images/#{flavor}/partner_sku_reviews/wc_theme_customizer_example.png", class: 'partner-sku-reviews-wc-example' %>
+
+### Admin panel
+
+Alternatively you can always manage the widgets from the admin panel.
+
+* Make sure you are logged in the admin panel and your user has a role that can edit the theme.
+* Navigate to `Appearance` and select `Widgets`.
+* A list with the available widget should appear on the left and the available widget areas on the right.
+* Drag and drop the widget from the left list to the widget area of your choice to the right.
+
+<%= image_tag "/assets/images/#{flavor}/partner_sku_reviews/wc_widget_admin_example.png", class: 'partner-sku-reviews-wc-example' %>
+
+## Add Widgets programmatically
+
+You can also add the widgets programmatically in any template by using the Wordpress provided [widget template tag](https://codex.wordpress.org/Function_Reference/the_widget).
+
+### Templates
+
+You could add the widgets in your [child theme's template](https://developer.wordpress.org/themes/advanced-topics/child-themes/) for the single product: `wp-content/themes/your child theme/woocommerce/content-single-product.php`. Create `woocommerce` directory if it doesn't exist in your child theme.
+
+#### Template tag for inline widget
+
+~~~php
+<?php the_widget( 'WC_Skroutz_Analytics_Product_Reviews_Inline_Widget' ); ?>
+~~~
+
+#### Template tag for extended widget
+
+~~~php
+<?php the_widget( 'WC_Skroutz_Analytics_Product_Reviews_Extended_Widget' ); ?>
+~~~
+
+### Ηοοks
+
+Instead of inserting the widget directly in the template you can use a [WooCommerce hook](https://docs.woocommerce.com/wc-apidocs/hook-docs.html) in your [child theme's](https://developer.wordpress.org/themes/advanced-topics/child-themes/) functions.php: `wp-content/themes/your child theme/functions.php`
+
+#### Hook for the inline widget
+
+Hook to display the inline widget after the add to cart button.
+Adjust hook priority, 35 in this example, according to your theme's needs
+~~~php
```

Comment:
I get this
![Screenshot from 2019-04-23 16-43-51](https://user-images.githubusercontent.com/6825357/56585870-4fab7c00-65e7-11e9-961d-7632f3407f37.png)

Reply:
Fixed it by adding a newline above `~~~php`